pkg/tfgen/docs.go

```diff
@@ -248,6 +263,66 @@ func getMarkdownDetails(sink diag.Sink, repoPath, org, provider string,
 	return markdownBytes, markdownFileName, true
 }
 
+// Create a regexp based replace rule that is bounded by non-ascii letter text.
+//
+// This function is not appropriate to be called in hot loops.
```

Comment:
You can lift `r` to `var boundedReplaceRegExp = ..` at the module level, removing the hot loops limitation.

Reply:
Ah never mind, it's staging the computation already.